packages/fxa-auth-server/lib/l10n/index.ts

```diff
@@ -0,0 +1,128 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { DOMLocalization, Localization } from '@fluent/dom';
+import { FluentBundle, FluentResource } from '@fluent/bundle';
+import { negotiateLanguages } from '@fluent/langneg';
+import { LocalizerBindings } from './bindings';
+import availableLocales from 'fxa-shared/l10n/supportedLanguages.json';
+import { EN_GB_LOCALES } from 'fxa-shared/l10n/otherLanguages';
+import { RendererBindings } from '../senders/renderer/bindings';
+
+type Bindings = LocalizerBindings | RendererBindings;
+export interface FtlIdMsg {
+  id: string;
+  message: string;
+}
+
+class Localizer {
+  protected readonly bindings: Bindings;
+
+  constructor(bindings: Bindings) {
+    this.bindings = bindings;
+  }
+
+  protected async fetchMessages(currentLocales: string[]) {
+    const fetched: Record<string, string> = {};
+    for (const locale of currentLocales) {
+      fetched[locale] = await this.fetchLocalizationMessages(locale);
+    }
+    return fetched;
+  }
+
+  protected createBundleGenerator(fetched: Record<string, string>) {
+    async function* generateBundles(currentLocales: string[]) {
+      for (const locale of currentLocales) {
+        const source = fetched[locale];
+        if (source) {
+          const bundle = new FluentBundle(locale, {
+            useIsolating: false,
+          });
+          const resource = new FluentResource(source);
+          bundle.addResource(resource);
+          yield bundle;
+        }
+      }
+    }
+
+    return generateBundles;
+  }
+
+  async setupLocalizer(acceptLanguage: string, needsDOM = false) {
```

Comment:
That's a good suggestion. I think this could use a little bit more work anyway since we probably don't need to localize (in the Node bindings anyway, I like using it for Storybook) if the `acceptLanguage` is `en` or `en-US`.